README.md

```diff
@@ -68,6 +68,7 @@ client
 | retryCondition | `Function` | `isNetworkOrIdempotentRequestError` | A callback to further control if a request should be retried.  By default, it retries if it is a network error or a 5xx error on an idempotent request (GET, HEAD, OPTIONS, PUT or DELETE). |
```

Comment:
Please extends this to add the function to say that receives two parameters (error, context) and should return a boolean otherwise it's going to be handled by the fallback